Ribbon/Readme.md

```diff
@@ -74,7 +74,7 @@ namespace Ribbon
 
 * The ribbon class derives from the `ExcelDna.Integration.CustomUI.ExcelRibbon` base class. This is how Excel-DNA itentifies the class a defining a ribbon controller.
 
-* The ribbon class must be 'COM visible'. Either the class must be marked as `[ComVisible(true)]` (the default class library template in Visual Studio markes the assembly as `[assembly:ComVisible(false)]`).
+* The ribbon class must be 'COM visible', to do so the class must be marked as `[ComVisible(true)]` (the default class library template in Visual Studio markes the assembly as `[assembly:ComVisible(false)]`).
```

Comment:
Sorry to leave this for so long. But every time I Iook at this, I don't know what to do. The intention is to say that Either the callss must be marked `[ComVisible(true)]`, or the assembly must be marked as `[assembly:ComVisible{true}]`, or the attribute on the assembly of `[assembly:ComVisible(false)]` that is added by the default project templates must be removed, reverting to the default state where the `ComVisible` state for the assembly is true.